.github/workflows/update-plugin-list.yml

```diff
@@ -27,12 +27,20 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: "3.8"
+          cache: pip
+      - name: requests-cache
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/pytest-plugin-list/
+          key: plugins-http-cache-${{ github.run_id }} # Can use time based key as well
+          restore-keys: plugins-http-cache-
```

Comment:
What's the reason to add `github.run_id` to the cache key instead of using a constant key?

Reply:
following https://github.com/actions/cache/blob/main/tips-and-workarounds.md#update-a-cache
a cachi is immutable, so the new version needs a new key